drift_dev/lib/src/writer/queries/query_writer.dart

```diff
@@ -218,11 +218,11 @@ class QueryWriter {
     if (update.needsAsyncMapping) {
       _buffer.write('Future.wait(rows.map(');
       _writeMappingLambda(update);
-      _buffer.write('))');
+      _buffer.write(')).toList()');
```

Comment:
I think we don't need it in this case since `Future.wait` takes an iterable and returns a list already.

Reply:
Oh yes that's true.